src/Status.php

```diff
@@ -69,35 +69,29 @@ public function getIndexNames()
     /**
      * Checks if the given index exists.
      *
-     * @param string $name Index name to check
-     *
      * @return bool True if index exists
      */
-    public function indexExists($name)
+    public function indexExists(string $name)
```

Comment:
It should not be a BC break as since PHP 7.2 and the parameter type widening  feature (https://wiki.php.net/rfc/parameter-no-type-variance) a subclass is allowed to widen the type, so that shouldn't break existing code overriding this method.